omnigibson/robots/robot_base.py

```diff
@@ -288,6 +288,7 @@ def _get_proprioception_dict(self):
             joint_qvel=joint_velocities,
             joint_qeffort=joint_efforts,
             robot_pos=pos,
+            robot_pose=np.append(pos[:2], ori[2]),
```

Comment:
this definitely seems a bit hacky. why is the pose only defined by (x,y,rz)?

Reply:
lets just expose 2d orientation as a separate dimension. pos is already just 3d pose with dim 2 dropped